assets/dev/js/editor/regions/panel/pages/elements/elements.js

```diff
@@ -160,6 +172,28 @@ PanelElementsLayoutView = Marionette.LayoutView.extend( {
 		this.categoriesCollection = categoriesCollection;
 	},
 
+	shouldNotAddWidget( widget ) {
```

Comment:
can this function check for positive ( having not in a function name is less readable )

Reply:
Done